test/e2e/fixture-builder.js

```diff
@@ -2078,9 +1969,11 @@ class FixtureBuilder {
   }
 
   build() {
-    this.fixture.meta = {
-      version: FIXTURE_STATE_METADATA_VERSION,
-    };
+    if (!this.fixture.meta) {
+      this.fixture.meta = {
+        version: FIXTURE_STATE_METADATA_VERSION,
+      };
+    }
```

Comment:
the onboarding fixture already has the version. We need to update this logic otherwise the version would be overwritten.